.github/workflows/ci.yaml

```diff
@@ -172,7 +172,6 @@ jobs:
   deploy-to-fleek:
     needs: [build]
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
```

Comment:
Oh just testing deployments on this particular branch

Reply:
name: CI
on:
  push:
    branches:
      - master

But seems like I also need to remove `master` here in order to get this branch deployed?

Reply:
Trying to reproduce 
<img width="1693" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/61768337/ec92c19c-a762-4d2c-83d8-5ad8c2931833">

Reply:
Yes, that makes sense. These commands should also work from your local. It would probably be quicker to iterate locally rather than waiting for the CI each time.

Reply:
> name: CI on: push: branches: - master
> 
> But seems like I also need to remove `master` here in order to get this branch deployed?

Nope, once build finished it showed up